dist/render.js

```diff
@@ -1,11 +1,14 @@
 "use strict";
-var __assign = (this && this.__assign) || Object.assign || function(t) {
-    for (var s, i = 1, n = arguments.length; i < n; i++) {
-        s = arguments[i];
-        for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
-            t[p] = s[p];
-    }
-    return t;
+var __assign = (this && this.__assign) || function () {
```

Comment:
Are you able to change this `var` ? Maybe change the function to an arrow function? Same question elsewhere that you've refactored.

Maybe it's a job for another PR?

Reply:
well, technically i did not refactor anything, that was the typescript compiler result, sure some internals changed. Maybe i need to tune something else.